webapp/src/components/backstage/playbook_runs/playbook_run_list/playbook_run_list.tsx

```diff
@@ -187,18 +197,16 @@ const BackstagePlaybookRunList = () => {
     }, [query]);
 
     useEffect(() => {
-        setFetchParams((oldParams) => {
-            return {...oldParams, team_id: currentTeam.id};
-        });
-    }, [currentTeam.id]);
+        const queryForTeamID = new URLSearchParams(query).get('team_id');
+        setFetchParams((oldParams) => ({...oldParams, status: queryForTeamID || ''}));
+    }, [query]);
 
     // When the component is first mounted (or the team changes), determine if there are any
```

Comment:
We should update this comment
```suggestion
    // When the component is first mounted, determine if there are any
```